rb/CHANGES

```diff
@@ -13,6 +13,7 @@ BiDi:
 
 Chrome:
 * Fix http proxy configuration for chrome (#13093)
+* Fully support Chrome 120+ old headless mode (#13271)
```

Comment:
Don't add to the changelog, we have an automated process that this will get in the way of.

Reply:
Oh, great!

Reply:
Done, I removed the commit that introduced this.